src/DIRAC/Resources/Computing/BatchSystems/Condor.py

```diff
@@ -19,50 +13,127 @@
 import os
 
 
+# Cannot use the PilotStatus module here as Condor is meant to be executed on a remote machine
+# DIRAC might not be available
+STATES_MAP = {
+    1: "Waiting",
+    2: "Running",
+    3: "Aborted",
+    4: "Done",
+    5: "Failed",
+}
+
+HOLD_REASON_SUBCODE = "55"
+
+subTemplate = """
+# Environment
+# -----------
+# There exist many universe:
+# https://htcondor.readthedocs.io/en/latest/users-manual/choosing-an-htcondor-universe.html
+universe = %(targetUniverse)s
+
+# Inputs/Outputs
+# --------------
+# Inputs: executable to submit
+executable = %(executable)s
+
+# Directory that will contain the outputs
+initialdir = %(initialDir)s
+
+# Outputs: stdout, stderr, log
+output = $(Cluster).$(Process).out
+error = $(Cluster).$(Process).err
+log = $(Cluster).$(Process).log
+
+# Transfer all output files, even if the job is failed
+transfer_output_files = ""
+should_transfer_files = YES
+when_to_transfer_output = ON_EXIT_OR_EVICT
```

Comment:
We moved them here because it should help us getting the outputs of the aborted pilots.
I guess we were using the vanilla options: https://htcondor.readthedocs.io/en/latest/users-manual/file-transfer.html?#specifying-what-files-to-transfer

Reply:
It "should" or it does?

Reply:
I checked again because I was not entirely sure with all the changes I made: it does if a pilot is removed.
I made a test by removing a `FAILED` pilot with `condor_rm`:
- `when_to_transfer_output = ON_EXIT_OR_EVICT`: I get the outputs
- `when_to_transfer_output = ON_EXIT`: 

```bash
Failed to find output for condor job htcondorce...
{'OK': False, 'Errno': 2, 'Message': 'No such file or directory ( 2 : Could not find ...'}
```

Reply:
Ok, good!